docs/language-server/protocol-language-server.md

```diff
@@ -1212,6 +1228,36 @@ interface Custom {
 }
 ```
 
+### `LibraryEntry`
+
+Represents a library available in a resolved edition.
```

Comment:
Maybe this description should briefly mention how additional metadata about a library can be retrieved. Just so that someone interested in how to, for example, get the library tag line is pointed in the right direction from here.

Reply:
Hmm, maybe the cached manifest could be used, but that mechanism is not yet implemented and was not originally meant to be exposed to the IDE, as it was just meant for the dependencies.

Canonically it should refer to the repository as described in repositories.md.

Reply:
> Canonically it should refer to the repository as described in repositories.md.
Then a link to there should be enough.